src/javascript/binary/pages/client/change_passwordws.js

```diff
@@ -0,0 +1,130 @@
+
+var PasswrdWS = (function(){
+
+
+	var validateForm = function() {
+
+		var errors = [];
+		var inputs = ['old', 'new', 'repeat'];
+		var $form = $("#changePasswordForm");
+
+		inputs.forEach(function(i){
+			value = $form.find("input."+i).val();
+
+			// Password too short
+			if(value.length < 5) {
+				
+			}
+
+			// Password too long
+			console.log(i);
+		});
+
+
+		var rules = {
+			minLength: {
+
+			}
+		};
+	};
+
+	var minLength = function(str) {
+
+	}
+
+	return {
+		validateForm: validateForm,
+		sendRequest: sendRequest
+	};
+})();
+
+function () { 
+	var bResult = true; 
+	var input_element_oldpassword = document.getElementById('oldpassword');
+	var error_element_erroroldpassword = document.getElementById('erroroldpassword');
+	document.getElementById('erroroldpassword').innerHTML = '';
+	if (input_element_oldpassword && error_element_erroroldpassword) {
+		var regexp;
+		var bInputResult = true;
+		regexp = new RegExp('^.+$');
+		if (bInputResult && !regexp.test(input_element_oldpassword.value)){
+			error_element_erroroldpassword.innerHTML = 'Please enter a password.';
```

Comment:
Will be using the standard `text.localize('Text');` as used elsewhere on Binary.com pages. By the way, the content of line 41 downward is a copy of inline javascript (inside form's `onsubmit`) from [the existing password change page](https://www.binary.com/user/change_password?l=EN); and will be refactored.